swift-sdk/Internal/EmbeddedMessagingManager.swift

```diff
@@ -16,16 +16,12 @@ class EmbeddedMessagingManager: NSObject, IterableEmbeddedMessagingManagerProtoc
         self.dateProvider = dateProvider
         
         super.init()
-        
-        initObservers()
-        initAutoFetchTimer()
     }
```

Comment:
I believe we are missing a start call here

Reply:
It's done in `InternalIterableAPI`. It was already getting called, but that was doing nothing before this PR.

Reply:
Oh gotcha at line 618